src/psyclone/tests/dynamo0p3_cma_test.py

```diff
@@ -47,8 +47,7 @@
 from psyclone.configuration import Config
```

Comment:
Add your name as 'modified by' at the top.

Reply:
I didn't see your name - if you decide not to include it, that's fine (I often find it 'wrong' to include me :) ), so I'll ignore it.